defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
```

Comment:
As long as the arguments are order independent I don't see a problem having a second variable that the casual user of this role does not touch. As soon as someone attempts to override these defaults the configuration potentially becomes incompatible with future role or certbot updates, which is why I would not recommend to override that variable from outside other than for testing purposes.

It's neat to have every flag in a different line, so that changes become easier to track in git.